### PR TITLE
Feature/single service view

### LINKED ISSUE #106: Create view and subscription for single service
Create a template and subscription that allows building single service summary view. Refer to requirements document for details on the view.

Definition of done
---
* [x] Single service summary template exists.
* [x] Template contains single service subscription.
* [ ] Template contains logic to show different content based on user status.
  * [ ] Where desirable, registered users and PO  see the same content.

### PR DESCRIPTION
**Pull request by frenchbread**
Closes #106 

# Proposed changes
- Added service template for single view
- Added breadcrumb
- Route

@Digipalvelutehdas/dipor-team please review.

Depends on #117.

### CODE REVIEW

**Comment by frenchbread** [comment]
This PR is `WIP`.

**Comment by frenchbread** [comment]
@Digipalvelutehdas/dipor-team Please review.

File: services/client/single/owners/owner/owner.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  // Get owner ID from template instance
+  instance.ownerId = instance.data.ownerId;
+
+  // Subscribe to service owner
+  instance.subscribe("serviceOwner", instance.ownerId);
+});
+
+Template.serviceOwner.helpers({
+  ownerEmail () {
```
Comment (brylie) [inline services/client/single/owners/owner/owner.js]: Nice use of ES2015 object literal :smiley:

File: services/client/single/owners/owner/owner.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    // Get reference to template instance
+    const instance = Template.instance();
+
+    // Get user by Id
+    const owner = Meteor.users.findOne(instance.ownerId);
+
+    // Make sure owner is available,
+    // since Meteor.users.findOne may return an empty array
```
Comment (brylie) [inline services/client/single/owners/owner/owner.js]: This comment seems uncorrect. `findOne()` can return undefined or a single object.

Your logic is fine, just the comment is misleading.

File: services/client/single/owners/owners.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      <span>
+        <i class="fa fa-users"></i>
+        {{_ "singleService-owners" }}
+      </span>
+    </div>
+    <div class="panel-body">
+      {{# if service.ownerIds }}
+      <ul class="list-group">
+        {{# each ownerId in service.ownerIds }}
+          {{> serviceOwner ownerId=ownerId }}
```
Comment (brylie) [inline services/client/single/owners/owners.html]: Great use of `{{each ... in ...}}` as well as named data context with `ownerId=ownerId`. :smile_cat:

**Comment by frenchbread** [comment]
@brylie Thanks! I fixed the comment.

**Pull request merged**